zetaclient/chains/solana/observer/inbound.go

```diff
@@ -298,7 +298,7 @@ func (ob *Observer) ParseInboundAsDeposit(
 	event := &clienttypes.InboundEvent{
 		SenderChainID: ob.Chain().ChainId,
 		Sender:        sender,
-		Receiver:      sender,
+		Receiver:      sender, // receiver is pulled out from memo
```

Comment:
This is not a question for this PR, but just a general note: I don't think we need a generic "inbound event" because we already have one — CCTX.